go/vt/sqlparser/ast.go

```diff
@@ -4944,6 +4944,7 @@ const (
 	NotRegexpStr         = "not regexp"
 	JSONExtractOp        = "->"
 	JSONUnquoteExtractOp = "->>"
+	AssignmentOpStr      = ":="
```

Comment:
(minor) It's a little misleading to put this constant in this section, just because the comment over the `const` section on line 4930 makes it seem like these are the valid `Operator` values for `ComparisonExpr`, but that's not the case for `AssignmentOpStr`. 

I would recommend moving this out of this block to a different spot in this file to avoid future code readers mistakenly thinking that... or... perhaps even simpler... it looks like this constant is only referenced in one place, in sql.y, so you could just change that line in sql.y to just return the literal `:=` value.